README.md

```diff
@@ -1,16 +1,18 @@
-# Gravitational Teleport
+# Teleport
 
-Gravitational Teleport is a modern security gateway for remotely accessing:
+Teleport is a modern security gateway for remotely accessing:
 * Clusters of Linux servers via SSH or SSH-over-HTTPS in a browser.
```

Comment:
Let's make all those links actually:

* [Clusters of Linux Servers](...) via SSH ...
* [Kuberntes Clusters](...)
* [Web applications](...)
* [Databases](preview link(

Reply:
Updated these links, once we've updated Teleport Server section I'll update the URL for the first 'cluster of linux servers' for now I' sent this to our quickstart